mmv1/products/compute/terraform.yaml

```diff
@@ -2100,6 +2100,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           health_check_name: "proxy-health-check"
           backend_name: "compute-backend"
           route_name: "route-ilb"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "route_ilb_vip"
+        primary_resource_id: "route-ilb"
+        min_version: beta
```

Comment:
I think this test probably still needs to be beta-only for now?

Reply:
Indeed. Is it possible to have an example that only shows in the beta docs?

Reply:
yeah, this min_version value will make it only show up in the beta docs.